iothub_client/src/iothubtransporthttp.c

```diff
@@ -1540,199 +1744,76 @@ static void DoEvent(HTTPTRANSPORT_HANDLE_DATA* handleData, HTTPTRANSPORT_PERDEVI
                         }
                         else
                         {
-                            /*Codes_SRS_TRANSPORTMULTITHTTP_17_078: [Every message property "property":"value" shall be added to the HTTP headers as an individual header "iothub-app-property":"value".] */
-                            MAP_HANDLE map = IoTHubMessage_Properties(message->messageHandle);
-                            const char*const* keys;
-                            const char*const* values;
-                            size_t count;
-                            if (Map_GetInternals(map, &keys, &values, &count) != MAP_OK)
+                            // set_message_properties returning false does not necessarily mean the the function failed, it just means
```

Comment:
>// set_message_properties returning false does not necessarily mean the the function failed, it just means [](start = 28, length = 106)

Hmm... if any of these calls fail, it feels we should just give up and log.  I don't necesarily think we retry on OOMs and other errors, like the message too long, aren't going to be helped by going through loop again.